core/src/main/resources/hudson/slaves/JNLPLauncher/main.jelly

```diff
@@ -51,13 +98,17 @@ THE SOFTWARE.
                 <p>
                   ${%Run from agent command line:}
                 </p>
-                <pre>javaws ${h.inferHudsonURL(request)}${it.url}slave-agent.jnlp</pre>
+                <textarea class="copiable" readonly="readonly">javaws ${h.inferHudsonURL(request)}${it.url}slave-agent.jnlp</textarea>
               </li>
               <li>
                 <p>
                   ${%Or if the agent is headless:}
                 </p>
-                <pre>java${it.launcher.vmargs == null ? '' : ' ' + it.launcher.vmargs} -jar <a href="${rootURL}/jnlpJars/slave.jar">slave.jar</a> -jnlpUrl ${h.inferHudsonURL(request)}${it.url}slave-agent.jnlp</pre>
+                  <textarea class="copiable" readonly="readonly"
+                            data-windows="Invoke-WebRequest -Uri $theurl -OutFile slave.jar\njava${it.launcher.vmargs == null ? '' : ' ' + it.launcher.vmargs} -jar slave.jar -jnlpUrl ${h.inferHudsonURL(request)}${it.url}slave-agent.jnlp"
+                            data-linux="curl -O ${h.inferHudsonURL(request)}jnlpJars/slave.jar\njava${it.launcher.vmargs == null ? '' : ' ' + it.launcher.vmargs} -jar slave.jar -jnlpUrl ${h.inferHudsonURL(request)}${it.url}slave-agent.jnlp">curl -O ${h.inferHudsonURL(request)}jnlpJars/slave.jar
```

Comment:
Yeah, I agree that from a technical standpoint, it's true. But I think putting POSIX instead of Linux would be less clear for a majority of users. Maybe Unix, yes, but well, it's probably also a minority of agents (pretty sure stats would back that btw).